FindOnPage.md

```diff
@@ -0,0 +1,382 @@
+    configuration.IsCaseSensitive = false;
+    configuration.ShouldMatchWord = false;
+    return configuration;
+}
+```
+
+### Start a Find Operation
+
+
+```cpp
+//! [StartFindOnPage]
+void AppWindow::StartFindOnPage(const std::wstring& findTerm)
+{
+    // Get the find interface
+    auto find = webView2Control.GetFind();
+
+    // Create find configuration
+    auto configuration = CreateFindConfiguration();
+
+    // Start the find operation
+    find.StartFind(configuration, OnFindOperationCompleted);
+}
+//! [StartFindOnPage]
+```
+### Stop an existing find operation
+#### Description
+To stop an ongoing find operation within a WebView2 control, developers can use the `StopFind` method of the `ICoreWebView2Find` interface.
+```cpp
+//! [StopFind]
+void AppWindow::StopFind()
+{
+    // Get the find interface
+    auto find = webView2Control.GetFind();
+    
+    // Stop the find operation
+    find.StopFind();
+}
+//! [StopFind]
+```
+
+### Retrieve the Number of Matches
+
+#### Description
+To retrieve the total number of matches found during a find operation within a WebView2 control, developers can utilize the `GetMatchCount` method.
+
+
+```cpp
+//! [GetMatchCount]
+void AppWindow::GetMatchCount()
+{
+    // Get the find interface
+    auto find = webView2Control.GetFind();
+
+    // Get the match count
+    LONG matchCount;
+    find.GetMatchesCount(&matchCount);
+
+    // Update UI or handle matchCount
+}
+//! [GetMatchCount]
+```
+
+### Retrieve the Index of the Active Match
+
+#### Description
+Developers can retrieve the index of the currently active match within a WebView2 control using the `GetActiveMatchIndex` method.
+
+
+```cpp
+//! [GetActiveMatchIndex]
+void AppWindow::GetActiveMatchIndex()
+{
+    // Get the find interface
+    auto find = webView2Control.GetFind();
+
+    // Get the active match index
+    LONG activeMatchIndex;
+    find.GetActiveMatchIndex(&activeMatchIndex);
+
+    // Update UI or handle activeMatchIndex
+}
+//! [GetActiveMatchIndex]
+```
+
+### Navigate to the Next Match
+
+#### Description
+To navigate to the next match found during a find operation within a WebView2 control, developers can use the `FindNext` method.
+
+
+```cpp
+//! [FindNext]
+void AppWindow::FindNext()
+{
+    // Get the find interface
+    auto find = webView2Control.GetFind();
+
+    // Find the next occurrence
+    find.FindNext();
+}
+//! [FindNext]
+```
+
+### Navigate to the Previous Match
+
+#### Description
+To navigate to the previous match found during a find operation within a WebView2 control, developers can use the `FindPrevious` method.
+
+
+```cpp
+//! [FindPrevious]
+void AppWindow::FindPrevious()
+{
+    // Get the find interface
+    auto find = webView2Control.GetFind();
+
+    // Find the previous occurrence
+    find.FindPrevious();
+}
+//! [FindPrevious]
+```
+
+#### Handle Match Count Changes
+
+```cpp
+void OnMatchCountChanged(LONG matchesCount)
+{
+    // Handle match count changes
```

Comment:
I just got rid of these, there doesn't seem to be a situation where the end dev would need to use any of these 3 handlers.

FindCompleted -> There is already a callback which handles relevant find completed tasks, no need for duplication.

MatchCountChanged -> This function is unnecessary because developers can simply call the GetMatchCount function provided by the API to retrieve the match count whenever they need it. There's no need for an event handler to specifically handle match count changes since it can be obtained through the getter function.

ActiveMatchIndexChanged -> Similarly, developers can retrieve the active match index using the GetActiveMatchIndex function provided by the API whenever they need it. There's no need for an event handler specifically for handling active match index changes. 

Additionally, in the case that they are **using their own UI** and want to keep track of whether the index changes or not, there is not a need for a event/handler because **the only thing that can modify the index is StartFind, FindNext, Find Previous, and technically StopFind** because it terminates the find session. Basically there is no opportunity for these variables to change unless explicitly directed to by the end dev. 

Reply:
Don't we need ActiveMatchIndexChanged and MatchCountChanged for telling the end dev when those change so the end dev can update their custom UI?